2025-03-03 06:20:16 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:20:16 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:20:17 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:30:00 [INFO]: Сейчас не время для отправки уведомлений (6:30). Разрешено с 9:00 до 21:00
2025-03-03 06:36:36 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:36:36 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:36:36 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:36:40 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:36:40 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:36:40 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:36:50 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:36:50 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:36:50 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:38:07 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:38:07 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:38:07 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:38:13 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 06:38:13 [INFO]: Модели синхронизированы с базой данных
2025-03-03 06:38:13 [INFO]: Планировщик уведомлений запущен
2025-03-03 06:45:00 [INFO]: Сейчас не время для отправки уведомлений (6:45). Разрешено с 9:00 до 21:00
2025-03-03 07:00:00 [INFO]: Запуск проверки новых заказов из Kaspi API
2025-03-03 07:00:00 [INFO]: Сейчас не время для отправки уведомлений (7:0). Разрешено с 9:00 до 21:00
2025-03-03 07:00:00 [ERROR]: Ошибка при получении заказов из Kaspi API: - Error: getaddrinfo ENOTFOUND kaspi-api-endpoint.kz
    at AxiosError.from (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:857:14)
    at RedirectableRequest.handleRequestError (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:3169:25)
    at RedirectableRequest.emit (node:events:536:35)
    at eventHandlers.<computed> (C:\Users\user\Desktop\bagaber-project\node_modules\follow-redirects\index.js:49:24)
    at ClientRequest.emit (node:events:524:28)
    at emitErrorEvent (node:_http_client:104:11)
    at TLSSocket.socketErrorListener (node:_http_client:512:5)
    at TLSSocket.emit (node:events:524:28)
    at emitErrorNT (node:internal/streams/destroy:170:8)
    at emitErrorCloseNT (node:internal/streams/destroy:129:3)
    at Axios.request (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:4258:41)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async KaspiService.fetchNewOrders (C:\Users\user\Desktop\bagaber-project\services\kaspiService.js:45:24)
    at async NotificationScheduler.checkNewOrders (C:\Users\user\Desktop\bagaber-project\services\notificationScheduler.js:43:22)
2025-03-03 07:00:00 [ERROR]: Ошибка при проверке новых заказов: - Error: Ошибка при получении заказов из Kaspi: getaddrinfo ENOTFOUND kaspi-api-endpoint.kz
    at KaspiService.fetchNewOrders (C:\Users\user\Desktop\bagaber-project\services\kaspiService.js:58:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async NotificationScheduler.checkNewOrders (C:\Users\user\Desktop\bagaber-project\services\notificationScheduler.js:43:22)
2025-03-03 07:02:24 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:02:24 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:02:24 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:02:32 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:02:32 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:02:32 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:04:35 [ERROR]: Ошибка подключения к базе данных RDS: - SequelizeHostNotFoundError: getaddrinfo ENOTFOUND your-rds-instance.region.rds.amazonaws.com
    at ConnectionManager.connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\mysql\connection-manager.js:96:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ConnectionManager._connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:222:24)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:174:32
    at async ConnectionManager.getConnection (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:197:7)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:305:26
    at async Sequelize.authenticate (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:457:5)
    at async testConnection (C:\Users\user\Desktop\bagaber-project\config\database.js:40:5)
    at async startServer (C:\Users\user\Desktop\bagaber-project\app.js:49:23)
2025-03-03 07:04:51 [ERROR]: Ошибка подключения к базе данных RDS: - SequelizeHostNotFoundError: getaddrinfo ENOTFOUND your-rds-instance.region.rds.amazonaws.com
    at ConnectionManager.connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\mysql\connection-manager.js:96:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ConnectionManager._connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:222:24)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:174:32
    at async ConnectionManager.getConnection (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:197:7)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:305:26
    at async Sequelize.authenticate (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:457:5)
    at async testConnection (C:\Users\user\Desktop\bagaber-project\config\database.js:40:5)
    at async startServer (C:\Users\user\Desktop\bagaber-project\app.js:49:23)
2025-03-03 07:05:06 [INFO]: Создан файл с базовыми шаблонами сообщений: C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:05:06 [ERROR]: Ошибка подключения к базе данных RDS: - SequelizeHostNotFoundError: getaddrinfo ENOTFOUND your-rds-instance.region.rds.amazonaws.com
    at ConnectionManager.connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\mysql\connection-manager.js:96:17)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async ConnectionManager._connect (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:222:24)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:174:32
    at async ConnectionManager.getConnection (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\dialects\abstract\connection-manager.js:197:7)
    at async C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:305:26
    at async Sequelize.authenticate (C:\Users\user\Desktop\bagaber-project\node_modules\sequelize\lib\sequelize.js:457:5)
    at async testConnection (C:\Users\user\Desktop\bagaber-project\config\database.js:40:5)
    at async startServer (C:\Users\user\Desktop\bagaber-project\app.js:95:25)
2025-03-03 07:22:11 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:22:12 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:22:12 [INFO]: Успешное подключение к базе данных
2025-03-03 07:22:13 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:22:13 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:22:13 [INFO]: Сервер запущен на порту 3001
2025-03-03 07:30:00 [INFO]: Сейчас не время для отправки уведомлений (7:30). Разрешено с 9:00 до 21:00
2025-03-03 07:45:00 [INFO]: Сейчас не время для отправки уведомлений (7:45). Разрешено с 9:00 до 21:00
2025-03-03 07:57:06 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:57:08 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:57:08 [INFO]: Успешное подключение к базе данных
2025-03-03 07:57:08 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:57:08 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:57:08 [INFO]: Сервер запущен на порту 3001
2025-03-03 07:57:11 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:57:12 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:57:12 [INFO]: Успешное подключение к базе данных
2025-03-03 07:57:13 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:57:13 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:57:13 [INFO]: Сервер запущен на порту 3001
2025-03-03 07:58:33 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:58:35 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 07:58:36 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 07:58:36 [INFO]: Успешное подключение к базе данных
2025-03-03 07:58:36 [INFO]: Модели синхронизированы с базой данных
2025-03-03 07:58:36 [INFO]: Планировщик уведомлений запущен
2025-03-03 07:58:36 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:00:00 [INFO]: Запуск проверки новых заказов из Kaspi API
2025-03-03 08:00:00 [INFO]: Запрашиваем заказы за период 1 дней
2025-03-03 08:00:00 [INFO]: Период разбит на 1 запросов
2025-03-03 08:00:00 [INFO]: Запрашиваем заказы с 2025-03-02T03:00:00.468Z по 2025-03-03T03:00:00.468Z
2025-03-03 08:00:00 [INFO]: Сейчас не время для отправки уведомлений (8:0). Разрешено с 9:00 до 21:00
2025-03-03 08:00:00 [ERROR]: Ошибка при получении заказов из Kaspi API: - AxiosError: Request failed with status code 404
    at settle (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:2031:12)
    at Unzip.handleStreamEnd (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:3148:11)
    at Unzip.emit (node:events:536:35)
    at endReadableNT (node:internal/streams/readable:1698:12)
    at process.processTicksAndRejections (node:internal/process/task_queues:90:21)
    at Axios.request (C:\Users\user\Desktop\bagaber-project\node_modules\axios\dist\node\axios.cjs:4258:41)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async KaspiService.fetchNewOrders (C:\Users\user\Desktop\bagaber-project\services\kaspiService.js:98:26)
    at async NotificationScheduler.checkNewOrders (C:\Users\user\Desktop\bagaber-project\services\notificationScheduler.js:43:22)
2025-03-03 08:00:00 [ERROR]: Ошибка при проверке новых заказов: - Error: Ошибка при получении заказов из Kaspi: Request failed with status code 404
    at KaspiService.fetchNewOrders (C:\Users\user\Desktop\bagaber-project\services\kaspiService.js:121:13)
    at process.processTicksAndRejections (node:internal/process/task_queues:105:5)
    at async NotificationScheduler.checkNewOrders (C:\Users\user\Desktop\bagaber-project\services\notificationScheduler.js:43:22)
2025-03-03 08:11:05 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:07 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:07 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:07 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:07 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:07 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:11:11 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:12 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:12 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:13 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:13 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:13 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:11:26 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:28 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:28 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:28 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:28 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:28 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:11:30 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:32 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:32 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:32 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:32 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:32 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:11:39 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:41 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:41 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:41 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:41 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:41 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:11:52 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:11:53 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:11:53 [INFO]: Успешное подключение к базе данных
2025-03-03 08:11:54 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:11:54 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:11:54 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:15:00 [INFO]: Сейчас не время для отправки уведомлений (8:15). Разрешено с 9:00 до 21:00
2025-03-03 08:17:15 [INFO]: Загружены шаблоны сообщений из файла C:\Users\user\Desktop\bagaber-project\config\templates.json
2025-03-03 08:17:16 [ERROR]: Ошибка при проверке статуса WhatsApp Cloud API: - [object Object]
2025-03-03 08:17:16 [ERROR]: Не удалось инициализировать WhatsApp Cloud API
2025-03-03 08:17:17 [INFO]: Успешное подключение к базе данных RDS
2025-03-03 08:17:17 [INFO]: Успешное подключение к базе данных
2025-03-03 08:17:17 [INFO]: Модели синхронизированы с базой данных
2025-03-03 08:17:17 [INFO]: Планировщик уведомлений запущен
2025-03-03 08:17:17 [INFO]: Сервер запущен на порту 3001
2025-03-03 08:27:35 [INFO]: Получен сигнал завершения работы, закрываем соединения...
